src/TestSuite/Stub/ConsoleOutput.php

```diff
@@ -59,7 +59,7 @@ public function write($message, int $newlines = 1)
             $newlines--;
         }
 
-        return false;
+        return 0;
```

Comment:
Should this be the length of the `$message`?

Reply:
Done

Reply:
Actually returning a non zero value causes tests for `ProgressHelper` tests to fail. This is a stub class (though why is it under `src` beats me) and seems to intentionally returns empty value to make testing easier.

Reply:
It is in src and it is part of the integration test system.